### PR TITLE
Fix CI rules not filtering integration test files properly

### PR DESCRIPTION
**Pull request by lengrongfu**
fixes https://github.com/containers/youki/issues/1639

### CODE REVIEW

**Comment by lengrongfu** [comment]
What is the wrong job in this pipeline?

**Comment by yihuaf** [comment]
> What is the wrong job in this pipeline?

It timed out after 15 min. I kick it again and see what happens.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -18,12 +18,18 @@ jobs:
...
-            tests/rust-integration-tests/integration_test: ./tests/rust-integration-tests/integration_test
-            tests/rust-integration-tests/test_framework: ./tests/rust-integration-tests/test_framework
+            crates/youki: 
+              - 'crates/youki/**'
+            crates/libcontainer: 
+              - 'crates/libcontainer/**'
+            crates/libcgroups: 
+              - 'crates/libcgroups/**'
+            tests/rust-integration-tests/runtimetest: 
+              - './tests/rust-integration-tests/runtimetest'
```
Comment (yihuaf) [inline .github/workflows/main.yml:28]:
```suggestion
              - './tests/rust-integration-tests/runtimetest/**'
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -18,12 +18,18 @@ jobs:
...
+            tests/rust-integration-tests/integration_test: 
+              - './tests/rust-integration-tests/integration_test'
```
Comment (yihuaf) [inline .github/workflows/main.yml:30]:
```suggestion
              - './tests/rust-integration-tests/integration_test/**'
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -18,12 +18,18 @@ jobs:
...
+            tests/rust-integration-tests/test_framework: 
+              - './tests/rust-integration-tests/test_framework'
```
Comment (yihuaf) [inline .github/workflows/main.yml:32]:
```suggestion
              - './tests/rust-integration-tests/test_framework/**'
```

**Review by yihuaf — CHANGES_REQUESTED** [review]

**Comment by yihuaf** [comment]
@lengrongfu I believe your change is good additions, but doesn't fix the underlying issue. I just checked this change with another PR I have and it doesn't trigger with integration test file changes. I believe the correct fix is in the review. Please make the change and I can merge :)

**Comment by yihuaf** [comment]
I apologize. I think I screwed up with directly merging the suggestions. Feel free to override my commit with the suggested change and force push again...

**Comment by lengrongfu** [comment]
I pass the test first, then review pr.

**Comment by yihuaf** [comment]
> I pass the test first, then review pr.

Can you elaborate what you mean?

**Comment by yihuaf** [comment]
> I will run effective tests to make sure this problem is fixed before going from draft to official pr.

I mean sure :) But I already tested for you. You can just include my suggested change (squash the commits) and then I can approve.

**Review by yihuaf — APPROVED** [review]
LGTM. Thank you.

**Pull request merged**